diagram-viewer/src/main/java/com/powsybl/diagram/viewer/MainViewController.java

```diff
@@ -43,6 +43,75 @@
  */
 public class MainViewController {
 
+    public enum ComponentFilterType {
+        ALL,
+        HVDC_LINE,
+        BUS,
```

Comment:
I think we can remove this enum, I was just trying how to make sense for the IdentifiableType BUS, but I don't think it's of interest to have that filtering. The opposite would be interesting though (no buses).